.circleci/config.yml

```diff
@@ -907,9 +907,11 @@ jobs:
           command: |
             if .circleci/scripts/test-run-e2e.sh
             then
-              MULTICHAIN=1 yarn test:e2e:chrome --retries 2 --debug
+              timeout 20m yarn test:e2e:chrome --retries 2 --debug
             fi
           no_output_timeout: 5m
+          environment:
+            MULTICHAIN: 1
```

Comment:
Oh cool you moved this to `environment`, that trick would have been useful on some things I tried in the past